Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: build clean run
+
+current_dir := $(shell pwd)
+current_user := $(shell id -u)
+
+build:
+	docker run --rm -v $(current_dir):/opt/accountapp/ -u $(current_user):$(current_user) -w /opt/accountapp --entrypoint ./gradlew java:8 --no-daemon --info war
+	docker-compose build run
+
+clean:
+	docker run --rm -v $(current_dir):/opt/accountapp/ -u $(current_user):$(current_user) -w /opt/accountapp --entrypoint ./gradlew java:8 clean
+	docker-compose rm -f run
+
+run:
```

Comment:
Personally, I prefer to keep them separated and run ```make build run```  so I clearly know what I am doing